packages/cli/__tests__/zosjobs/__unit__/view/spool-file-by-id/SpoolFileById.handler.unit.test.ts

```diff
@@ -35,6 +35,7 @@ describe("zos-jobs view spool-file-by-id handler", () => {
 
     afterEach(() => {
         jest.resetAllMocks();
+        DEFAULT_PARAMETERS.arguments.encoding = undefined; // Why is this needed? Where are things being set???
```

Comment:
Is this a todo comment or is it ok to leave for now? 😋

Reply:
I figured it out, it was because we weren't deep cloning objects, but were instead assigning them to new objects. The sub-objects were being passed by reference and polluting the results of other tests. We might want to look into any instances where we are using `Object.assign({},` in the future.